server/storage/main.lua

```diff
@@ -24,7 +24,7 @@ local groups = require 'server.storage.groups'
 ---@field fetchIsUnique fun(type: UniqueIdType, value: string|number): boolean
 ---@field addPlayerToJob fun(citizenid: string, group: string, grade: integer)
 ---@field addPlayerToGang fun(citizenid: string, group: string, grade: integer)
----@field fetchPlayerGroups fun(citizenid: string): table<string, Job>, table<string, Gang>
+---@field fetchPlayerGroups fun(citizenid: string): table<string, integer>, table<string, integer> jobs, gangs
```

Comment:
Why this change and why the jobs, gangs at the end?

Reply:
Trying to communicate that the first table are jobs, and the second table are gangs. I guess the non lazy way to do it is with aliases.

This is a fix to make the documentation match the actual return value of the function